tests/evhtests/l7_evh_ingressclass_test.go

```diff
@@ -331,6 +332,95 @@ func TestEVHAviInfraSettingNamingConvention(t *testing.T) {
 		t.Fatalf("Couldn't DELETE the Ingress %v", err)
 	}
 	integrationtest.DeleteSecret(secretName, ns)
+	integrationtest.RemoveAnnotateAKONamespaceWithInfraSetting(t, ns)
+	integrationtest.TeardownAviInfraSetting(t, settingName)
+	TearDownTestForIngress(t, modelName, settingModelName)
+	integrationtest.TeardownIngressClass(t, ingClassName)
+	waitAndVerify(t, ingClassName)
+	VerifyEvhNodeDeletionFromVsNode(g, modelName, vsKey, evhKey)
+}
+
+// AviInfraSetting CRD
+func TestEVHAviInfraSettingPerNSNamingConvention(t *testing.T) {
+	if lib.VIPPerNamespace() {
```

Comment:
Why are we skipping for VIPPerNamespace?

Reply:
For VIPPerNS, the naming conventions are different, it's the same in UT above this